pkg/collector/corechecks/containers/docker/check_test.go

```diff
@@ -37,6 +37,8 @@ func TestDockerCheckGenericPart(t *testing.T) {
 		// Should never been called as we are in the Docker check
 		generic.CreateContainerMeta("containerd", "cID101"),
 	}
+	fakeTagger := fxutil.Test[tagger.Mock](t, tagger.MockModule())
```

Comment:
open question: should we create a function to avoid to write this error prone line? 

```golang
// package utils 
//...
func NewFakeTagger() tagger.Mock {
  return fxutil.Test[tagger.Mock](t, tagger.MockModule())
}
```

```suggestion
	fakeTagger := taggerTestUtils.NewFakeTagger()
```